packages/app-mobile/tools/buildInjectedJs.ts

```diff
@@ -7,7 +7,7 @@ import { mkdirp, readFile, writeFile } from 'fs-extra';
 import { dirname, extname, basename } from 'path';
 const execa = require('execa');
 
-import webpack from 'webpack';
+import * as webpack from 'webpack';
```

Comment:
We want this to be compiled to `const webpack = require('webpack');`. As such, a namespace import is done here.

See also [the TypeScript documentation.](https://www.typescriptlang.org/tsconfig#esModuleInterop)